src/sandstorm/run-bundle.c++

```diff
@@ -1183,18 +1189,17 @@ private:
     KJ_SYSCALL(sigprocmask(SIG_SETMASK, &sigset, nullptr));
   }
 
-  void copyEtc() {
+  void linkEtc() {
     auto files = splitLines(readAll("etc.list"));
```

Comment:
Oh, do you have resolvconf installed?  can you look to see if /etc/resolv.conf is a symlink on your machine, and if it is, tell me what it points to?